.github/workflows/ci.yml

```diff
@@ -11,13 +10,6 @@ jobs:
       - run: npm i
       - run: npm run lint
 
-  typecheck:
```

Comment:
Why not typecheck the root project too? The files are all .js but it's still nice to check for bugs and use jsdoc types.

Reply:
I had an issue: `tsc` in the CI could not recognize an import from @hyparam/components, while it works locally - as I was not sure it was important, and as it was hard to reproduce, I just removed the typescript check. I'll try to make it work.

Reply:
OK, I added it again. The issue was that... I had to build the dependency first before being able to run tsc.